scripts/libraries/urequests.py

```diff
@@ -171,8 +171,9 @@ def request(method, url, data=None, json=None, files=None, headers={}, auth=None
         resp_headers = b"\r\n".join(resp_headers)
         content = b"\r\n".join(response)
     except OSError:
-        s.close()
```

Comment:
I think you should keep this line too, if there's an exception it will close the socket, not sure if finally is executed after `raise`

Reply:
Actually it is, never mind.